CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to shiplift
+Contributing to shiplift isn't limited to just filing bugs, users are more than welcomed to make suggestions, report any issue they may find, and make pull requests to help make shiplift better.
+
+## Working on shiplift
+### Prerequisites
+* [Git](https://git-scm.com/)
+
+### Getting shiplift
+1. Fork a copy of our repo
+2. Open up Git in an environment of your choice
+3. Run the following
+
+```
+$ git clone https://github.com/YOUR-GITHUB-PROFILE-NAME/shiplift.git
+$ cd shiplift
+```
+
+### Please pay attention to
+1. open an issue describing the feature/bug you wish to contribute first to start a discussion, explain why, what and how
+2. follow the eslint rules
+3. write tests covering 100% of the library code you produce
+4. one PR per feature/fix unless you follow [standard-version](https://github.com/conventional-changelog/standard-version) commit guidelines
```

Comment:
I'm going to opt out of making commit message guidelines for now. I've found that while useful for maintainers, its not a very good experience for contributors. Instead I'm just asking for notes/hightlights for the changelog in the pull request description.

Reply:
ok, changed